datadog_checks_dev/datadog_checks/dev/tooling/commands/meta_commands/prometheus.py

```diff
@@ -0,0 +1,279 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+from collections import defaultdict
+from itertools import product
+
+import click
+import pyperclip
+import requests
+from six import iteritems, itervalues
+from six.moves import input, range
+
+from ..utils import CONTEXT_SETTINGS, abort, echo_info, echo_success, echo_waiting, echo_warning
+from ...constants import get_root
+from ....utils import dir_exists, ensure_unicode, path_join, write_file_lines
+
+METRIC_SEPARATORS = ('.', '_')
+TYPE_MAP = {
+    'gauge': 'gauge',
+    'counter': 'count',
+    'histogram': 'gauge',
+}
+
+
+def sanitize_endpoint(endpoint):
+    if not endpoint.startswith('http'):
+        endpoint = 'http://{}{}'.format(
+            'localhost' if endpoint.startswith(':') else '',
+            endpoint
+        )
+
+    return endpoint
+
+
+def parse_metrics(endpoint):
+    metrics = defaultdict(dict)
+    response = requests.get(endpoint, stream=True)
+
+    for line in response.iter_lines(decode_unicode=True):
+        if line.startswith('#'):
+            try:
+                _, info_type, metric, info_value = line.split(' ', 3)
+            except ValueError:
+                continue
```

Comment:
Can you include an example or link of what one of these lines look like? I think that'd help for code readability and quickly be able to update/spot issues if something changes.